### DISCUSSION TITLE
Transparent grayscale png to svg conversion

### DISCUSSION THREAD

**Discussion by abhimanyu891998** [visioncortex/vtracer#81] [Q&A] [answered]
I have a png with no background, it is a grayscale image. When I try to convert it to SVG, the colors get inverted, and the background becomes black. Is there a way to preserve the transparent background and possibly not invert the colors of the image? I have attached the image for the reference.

![boywithwandbgremove](https://github.com/visioncortex/vtracer/assets/20404572/c6673507-c8d1-4ef6-8cf3-2122f7227a00)

**Comment by tyt2y3** [comment]
You must be referring to the webapp. Sadly, the original intention to invert the black and white is to make the effect more visible. Looking back, I think it is a bad idea. If you use the cmd app, it won't have this strange behaviour.

**Comment by tyt2y3** [comment, marked as answer]
I updated the webapp! however your image is not strictly black & white. here is the parameters that works for me:
![image](https://github.com/visioncortex/vtracer/assets/1782664/260802af-d386-4543-b075-5f5ca7acd4b9)

**Reply by abhimanyu891998** [reply]
Hey! Thanks a lot for your answer :)
